libs/platform/src/lib/components/value-help-dialog/value-help-dialog/value-help-dialog.component.html

```diff
@@ -276,26 +280,38 @@ <h5 fd-title>{{ searchField.advancedSearchLabel }}</h5>
 </ng-template>
 
 <ng-template #mobileTabs>
-    <ul fd-list [compact]="true" *ngIf="!hasSelectedTab && showSelectionTab && showDefineTab">
-        <li fd-list-item *ngIf="showSelectionTab">
-            <a fd-list-link (click)="switchTab(_tabTypes.selectFromList)">
-                <span fd-list-title>{{ selectTabTitle }}</span>
-                <i fd-list-icon glyph="navigation-right-arrow"></i>
-            </a>
-        </li>
-        <li fd-list-item *ngIf="showDefineTab">
-            <a fd-list-link (click)="switchTab(_tabTypes.defineConditions)">
-                <span fd-list-title>{{ defineTabTitle }}</span>
-                <i fd-list-icon glyph="navigation-right-arrow"></i>
-            </a>
-        </li>
-    </ul>
+  <div *ngIf="!hasSelectedTab && showSelectionTab && showDefineTab">
+    <div fd-bar [inPage]="true"  *ngIf="showSelectionTab">
+      <div fd-bar-left>
+        <fd-bar-element>
+            {{ selectTabTitle }}
+        </fd-bar-element>
+      </div>
+    <div fd-bar-right>
+        <fd-bar-element>
+			<fd-button-bar glyph="navigation-right-arrow" (click)="switchTab(_tabTypes.selectFromList)" title="{{selectTabTitle}} tab "></fd-button-bar>
+        </fd-bar-element>
```

Comment:
For better readability:

```suggestion
        <fd-bar-element>
		<fd-button-bar 
			glyph="navigation-right-arrow" 
			title="{{selectTabTitle}} tab "
			(click)="switchTab(_tabTypes.selectFromList)">
		</fd-button-bar>
        </fd-bar-element>
```